Modern/wrapperPlugins/src/test/java/org/bonej/wrapperPlugins/FractalDimensionWrapperTest.java

```diff
@@ -94,9 +94,9 @@ public void testPointResults() throws Exception {
 			.toArray();
 		final double[][] expectedCounts = { emptyCounts, cubeCounts, cubeCounts,
 			emptyCounts };
-		final String[] expectedLabels = { "Test Channel: 1, Time: 1",
-			"Test Channel: 2, Time: 1", "Test Channel: 1, Time: 2",
-			"Test Channel: 2, Time: 2" };
+//		final String[] expectedLabels = { "Test Channel: 1, Time: 1",
+//			"Test Channel: 2, Time: 1", "Test Channel: 1, Time: 2",
+//			"Test Channel: 2, Time: 2" };
 		final ImgPlus<BitType> imgPlus = createTestHyperStack("Test");
```

Comment:
Please don't leave commented out code in. If we need it again, we'll find it in an older commit